pkg/arvo/sys/vane/ames.hoon

```diff
@@ -1434,6 +1516,57 @@
     ^+  event-core
     %-  %^  trace  odd.veb  sndr.packet
         |.("received packet")
+    :: add the ship to our sponsees map if we are the sponsor
+    ::
+    =*  relay-state  relay-state.ames-state
+    =?  sponsees.relay-state
+      ?&
+        =(~ next.packet)
+        =/  val  (~(get by sponsees.relay-state) sndr.packet)
+        =(~ val)
+        =(our (sein:title our now sndr.packet))
+      ==
+      ::  send 30s expiration timer to behn
+      ::
+      =/  duct  ~[/ames]
+      =/  wire  (make-sponsee-timer-wire sndr.packet)
+      =.  event-core
```

Comment:
This mutation won't escape the enclosing `=?`.  You'll need to `=?  event-core`, I think.